set.mm

```diff
@@ -226317,6 +226550,16 @@ operation is a permutation group (group consisting of permutations), see
       L $.
   $}
 
+  ${
+    odcld.1 $e |- B = ( Base ` G ) $.
+    odcld.2 $e |- O = ( od ` G ) $.
+    odcld.3 $e |- ( ph -> A e. B ) $.
+    $( Deduction associated with ~ odcl .  (Contributed by Rohan Ridenour,
```

Comment:
Comment: "The order of a group element is always a nonnegative integer, deduction form."

Reply:
Used your comment suggestion.